input-source/domains/public-domain-additions.txt

```diff
@@ -1,2 +1,3 @@
 glthubs.com
 ukreturnclaim.com
+http://copyirghtagencymediacenter.ml
```

Comment:
you should not append the protocol, only the domain `copyirghtagencymediacenter.ml`